Game/MyProject/Game/Test/TestHelper/TestHelper.cpp

```diff
@@ -0,0 +1,91 @@
+//-------------------------------------------------------------------------------
+//-- TestHelper.cpp -------------------------------------------------------------
+//-------------------------------------------------------------------------------
+
+//-------------------------------------------------------------------------------
+//-- Dependencies ---------------------------------------------------------------
+//-------------------------------------------------------------------------------
+#include "TestHelper.h"
+
+void
+TestHelper::Setup()
+{
+    helperClass=std::make_shared<HelperClass>();
+    helper=std::make_shared<HelperManager>();
+    
+    boxMesh=helper->CreateBoxMesh();
+    quadMesh=helper->CreateQuadMesh();
+    sphereMesh=helper->CreateSphereMesh();
+    
+    createdBoxes.push_back(false);
+    createdBoxes.push_back(false);
+    
+    createdQuads.push_back(false);
+    createdQuads.push_back(false);
+    createdQuads.push_back(false);
+   
+    
+}
+
+void
+TestHelper::Draw()
+{
+//    //test boxes
```

Comment:
don't commit blocks of commented code.
if you want to keep it around for later testing move it to a function and just don't call it right now..